index.html

```diff
@@ -37,7 +37,7 @@
           <ul>
             <li class="tab" data-zone="entries"><a href="/" class="icon-briefcase">Portfolio</a></li>
             <li class="tab" data-zone="about"><a href="/about" class="icon-svg">About</a></li>
-            <li><a href="http://github.com/lexm" target="_blank">GitHub</a></li>
+            <li class="tab" data-zone="repos"><a href="/repos">GitHub</a></li>
```

Comment:
You may want to consider still including a link directly to your Github profile along with your repos. Most people expect a Github link to navigate directly to Github.